Packs/Lumu/Integrations/Lumu/Lumu.yml

```
@@ -0,0 +1,1031 @@
+      description: ""
+      type: Date
+    - contextPath: Lumu.RetrieveClosedIncidents.items.statusTimestamp
+      description: ""
+      type: Date
+    - contextPath: Lumu.RetrieveClosedIncidents.items.status
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveClosedIncidents.items.contacts
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveClosedIncidents.items.adversaries
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveClosedIncidents.items.adversaryId
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveClosedIncidents.items.adversaryTypes
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveClosedIncidents.items.description
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveClosedIncidents.items.labelDistribution.37
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveClosedIncidents.items.totalEndpoints
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveClosedIncidents.items.lastContact
+      description: ""
+      type: Date
+    - contextPath: Lumu.RetrieveClosedIncidents.items.unread
+      description: ""
+      type: Boolean
+    - contextPath: Lumu.RetrieveClosedIncidents.paginationInfo.page
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveClosedIncidents.paginationInfo.items
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveClosedIncidents.paginationInfo.next
+      description: ""
+      type: Number
+    description: Get a paginated list of closed incidents for the company. The items are listed by the most recent.
+  - name: lumu-retrieve-endpoints-by-incident
+    arguments:
+    - name: lumu_incident_id
+      description: ""
+    - name: page
+      description: ""
+    - name: items
+      description: ""
+    outputs:
+    - contextPath: Lumu.RetrieveEndpointsByIncident.items.label
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveEndpointsByIncident.items.endpoint
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveEndpointsByIncident.items.total
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveEndpointsByIncident.items.first
+      description: ""
+      type: Date
+    - contextPath: Lumu.RetrieveEndpointsByIncident.items.last
+      description: ""
+      type: Date
+    - contextPath: Lumu.RetrieveEndpointsByIncident.paginationInfo.page
+      description: ""
+      type: Number
+    - contextPath: Lumu.RetrieveEndpointsByIncident.paginationInfo.items
+      description: ""
+      type: Number
+    description: |-
+      Get a paginated summary of the endpoints affected by a specified incident.
+
+      | `{incident-uuid}` | uuid of the specific incident |
+      |---|---|
+  - name: lumu-mark-incident-as-read
+    arguments:
+    - name: lumu_incident_id
```

this argument is mandatory, please update the all the relevant arguments also. 

```suggestion
    - name: lumu_incident_id
      required: true
```

---

required flag added in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7